src/pages/STG_PanelAddrVerification.page

```diff
@@ -5,17 +5,17 @@
 
         var settingsHelpText = {};
 
-        var hideUnusedElements = function() {
+        var hideUnusedElements = function () {
 
             //drop-down selection, if in Edit mode
             var selection = j$('[id$=iClass__c] :selected').text();
 
             //existing value, if in View mode
-            if(!selection) {
+            if (!selection) {
                 selection = j$('[id$=iServiceName]').text();
             }
 
-            if(selection=='SmartyStreets') {
+            if (selection == 'SmartyStreets') {
```

Comment:
If you're adding spaces to these, should these also be changed to `===`?

Reply:
I can totally disable that feature in VSCode, I actually like the way it cleans up the spacing but if you would prefer to turn it off I can do that and resubmit without formatting. Or maybe we turn on the auto formatting and just have a cleanup PR? Please let me know if it's ok for this PR or if you would like it changed.

Reply:
It's just that it makes it look like huge blocks of code were changed in the file, when it may just be a few areas. I do like the idea of a separate cleanup PR, but we'd have to make sure that it wouldn't get overridden by other IDE's and update our docs to state that all VF files should be formatted this way, as well as document how to do this in IntelliJ as well as VSCode.